modules/installation-initializing.adoc

```diff
@@ -201,6 +205,14 @@ ifndef::rhv[]
 For production {product-title} clusters on which you want to perform installation debugging or disaster recovery, specify an SSH key that your `ssh-agent` process uses.
 ====
 endif::rhv[]
+ifdef::alibaba[]
+... Select *Alibaba* as the platform to target.
+... If you do not have an Amazon Web Services (AWS) profile stored on your computer, enter the AWS
+access key ID and secret access key for the user that you configured to run the
+installation program.
+... Select the AWS region to deploy the cluster to.
+... Select the base domain for the Route 53 service that you configured for your cluster.
+endif::alibaba[]
```

Comment:
@mtulio @jianli-wei @kwoodson @joelddiaz
Can we get the installer prompts? Or, if I can get an Alibaba installer, I can get the prompts.

Reply:
I ran it locally:
```
./bin/openshift-install create install-config --dir test
? SSH Public Key /home/kwoodson/.ssh/openshift_id_rsa.pub
? Platform alibabacloud
? Region us-east-1
? Base Domain alicloud-dev.devcluster.openshift.com
? Cluster Name demo
? Pull Secret [? for help]
INFO Install-Config created in: test
```